tests broken on Windows with latest Luarocks

@hishamhm could you take a look at https://ci.appveyor.com/project/hishamhm/hererocks/builds/35418067/job/s6k82innxl02vnf4

```
AssertionError: Error running command 'test\here\bin\luarocks.bat --version': code 255, output:
b'The syntax of the command is incorrect.\r\n'
```

--- a/README.rst
+++ b/README.rst
@@ -127,7 +127,7 @@ Luarocks supports RaptorJIT since 3.2.0.
 Installing LuaRocks
 ^^^^^^^^^^^^^^^^^^^
 
-Available versions: 2.0.8 - 2.4.4, 3.0.0 - 3.4.0. ``latest`` and ``^`` version aliases point to ``3.4.0``.
+Available versions: 2.0.8 - 2.4.4, 3.0.0 - 3.5.0. ``latest`` and ``^`` version aliases point to ``3.5.0``.
 
 Version 2.0.8 does not support Lua 5.2. Versions 2.0.8 - 2.1.2 do not support Lua 5.3.
 

--- a/hererocks.py
+++ b/hererocks.py
@@ -1899,7 +1899,8 @@ class LuaRocks(Program):
         "3.1.0", "3.1.1", "3.1.2", "3.1.3",
         "3.2.0", "3.2.1",
         "3.3.0", "3.3.1",
-        "3.4.0"
+        "3.4.0",
+        "3.5.0",
     ]
     translations = {
         "2": "2.4.4",
@@ -1914,8 +1915,9 @@ class LuaRocks(Program):
         "3.2": "3.2.1",
         "3.3": "3.3.1",
         "3.4": "3.4.0",
-        "^": "3.4.0",
-        "latest": "3.4.0"
+        "3.5": "3.5.0",
+        "^": "3.5.0",
+        "latest": "3.5.0"
     }
     checksums = {
         "luarocks-2.0.10.tar.gz"   : "11731dfe6e210a962cb2a857b8b2f14a9ab1043e13af09a1b9455b486401b46e",
@@ -1982,6 +1984,8 @@ class LuaRocks(Program):
         "luarocks-3.3.1-win32.zip" : "5ba3f6034e26e0bf93145354dca6aadb0bd7ad85c85e603a323787273db508bb",
         "luarocks-3.4.0.tar.gz"    : "62ce5826f0eeeb760d884ea8330cd1552b5d432138b8bade0fa72f35badd02d0",
         "luarocks-3.4.0-win32.zip" : "cd2b26d34f36cfbf69e3d80a8d25c46529e367f0866c41ac96049081a5653fc0",
+        "luarocks-3.5.0.tar.gz"    : "701d0cc0c7e97cc2cf2c2f4068fce45e52a8854f5dc6c9e49e2014202eec9a4f",
+        "luarocks-3.5.0-win32.zip" : "69a225f339a96cdbfe8eb5fcfb108c2eef270dab172bc09e78190a8a0932ffc8",
     }
 
     def get_download_name(self):
@@ -2410,8 +2414,8 @@ def main(argv=None):
     parser.add_argument(
         "-r", "--luarocks", help="Version of LuaRocks to install. "
         "Version can be specified in the same way as for standard Lua. "
-        "Versions 2.0.8 - 3.4.0 are supported. "
-        "'latest' and '^' are aliases for 3.4.0. "
+        "Versions 2.0.8 - 3.5.0 are supported. "
+        "'latest' and '^' are aliases for 3.5.0. "
         "Default git repo is https://github.com/luarocks/luarocks. "
         "Note that Lua 5.2 is not supported in LuaRocks 2.0.8, "
         "Lua 5.3 is supported only since LuaRocks 2.2.0, Lua 5.4 is supported only since "

